Acme.Interface.WebAPI/Controllers/v1/BooksController.cs

```diff
@@ -30,16 +31,23 @@ public BooksController(IBooksApiService booksApiService)
     /// <summary>
     /// Gets all books paged asynchronously.
     /// </summary>
-    /// <param name="pagingDto">The paged request DTO.</param>
+    /// <param name="page">The page.</param>
+    /// <param name="pageSize">Size of the page.</param>
+    /// <param name="sortColumn">The sort column.</param>
+    /// <param name="sortDirection">The sort direction.</param>
     /// <returns>
     /// Paged books.
     /// </returns>
-    [HttpPost("search")]
+    [HttpGet("search")]
```

Comment:
This endpoint should be at the root of the controller:
```
...api/v1/books
```
instead of 
```
...api/v1/books/search
```
So just clear out the parameters.